module/PowerShellEditorServices/Commands/PowerShellEditorServices.Commands.psd1

```diff
@@ -77,7 +77,8 @@ FunctionsToExport = @('Register-EditorCommand',
                       'Out-CurrentFile',
                       'Join-ScriptExtent',
                       'Test-ScriptExtent',
-                      'psedit')
+                      'psedit',
```

Comment:
There is an `AliasesToExport` field down below to put `psedit` in.  That is where it should go.

Reply:
Doh! Fixed now.